{"repository":"https://github.com/nextdoor/zkwatcher.orig","dockerfilePath":"Dockerfile","startTime":1672423862862,"clone":{"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/nextdoor/zkwatcher.orig'...\n","error":null,"commit":"65d848e6939356a0224f082593668b4ed2d9d4f5"},"originalBuild":{"startTimestamp":1672423863736,"endTimestamp":1672423899838,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t zkwatcher.orig:latest -f /tmp/dinghy-analysis/nextdoor/zkwatcher.orig/Dockerfile ."},"stdout":"Sending build context to Docker daemon  197.6kB\r\r\nStep 1/13 : FROM alpine:3.11.2\n3.11.2: Pulling from library/alpine\ne6b0cf9c0882: Pulling fs layer\ne6b0cf9c0882: Verifying Checksum\ne6b0cf9c0882: Download complete\ne6b0cf9c0882: Pull complete\nDigest: sha256:2171658620155679240babee0a7714f6509fae66898db422ad803b951257db78\nStatus: Downloaded newer image for alpine:3.11.2\n ---> cc0abc535e36\nStep 2/13 : MAINTAINER Matt Wise <matt@nextdoor.com>\n ---> Running in 4b31ca0064da\nRemoving intermediate container 4b31ca0064da\n ---> ee7533698d7d\nStep 3/13 : RUN apk add --update bash python3 ca-certificates curl wget\n ---> Running in 57f11c9a66b6\nfetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz\nfetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz\n(1/16) Installing ncurses-terminfo-base (6.1_p20200118-r4)\n(2/16) Installing ncurses-libs (6.1_p20200118-r4)\n(3/16) Installing readline (8.0.1-r0)\n(4/16) Installing bash (5.0.11-r1)\nExecuting bash-5.0.11-r1.post-install\n(5/16) Installing ca-certificates (20191127-r2)\n(6/16) Installing nghttp2-libs (1.40.0-r1)\n(7/16) Installing libcurl (7.79.1-r0)\n(8/16) Installing curl (7.79.1-r0)\n(9/16) Installing libbz2 (1.0.8-r1)\n(10/16) Installing expat (2.2.9-r1)\n(11/16) Installing libffi (3.2.1-r6)\n(12/16) Installing gdbm (1.13-r1)\n(13/16) Installing xz-libs (5.2.4-r0)\n(14/16) Installing sqlite-libs (3.30.1-r2)\n(15/16) Installing python3 (3.8.10-r0)\n(16/16) Installing wget (1.20.3-r0)\nExecuting busybox-1.31.1-r8.trigger\nExecuting ca-certificates-20191127-r2.trigger\nOK: 71 MiB in 30 packages\nRemoving intermediate container 57f11c9a66b6\n ---> f69d950aa802\nStep 4/13 : RUN mkdir /app /app/zk_watcher\n ---> Running in cee4bc611be4\nRemoving intermediate container cee4bc611be4\n ---> 60af92a639f7\nStep 5/13 : ADD requirements*.txt /app/\n ---> 3f6ebf53f2b3\nStep 6/13 : RUN pip3 install -r /app/requirements.test.txt -r /app/requirements.txt\n ---> Running in 206de030354a\nCollecting nose\n  Downloading nose-1.3.7-py3-none-any.whl (154 kB)\nCollecting mock\n  Downloading mock-5.0.0-py3-none-any.whl (29 kB)\nCollecting pep8\n  Downloading pep8-1.7.1-py2.py3-none-any.whl (41 kB)\nCollecting pyflakes\n  Downloading pyflakes-3.0.1-py2.py3-none-any.whl (62 kB)\nCollecting coverage\n  Downloading coverage-7.0.1.tar.gz (781 kB)\n  Installing build dependencies: started\n  Installing build dependencies: finished with status 'done'\n  Getting requirements to build wheel: started\n  Getting requirements to build wheel: finished with status 'done'\n  Installing backend dependencies: started\n  Installing backend dependencies: finished with status 'done'\n    Preparing wheel metadata: started\n    Preparing wheel metadata: finished with status 'done'\nCollecting Sphinx\n  Downloading sphinx-6.0.0-py3-none-any.whl (3.0 MB)\nCollecting nd_service_registry>=1.0.0\n  Downloading nd_service_registry-1.1.0.tar.gz (44 kB)\n\u001b[91m    ERROR: Command errored out with exit status 1:\n     command: /usr/bin/python3.8 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-xgj4jp82/nd-service-registry_bbc930685b444ab7b382e919df21c501/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-xgj4jp82/nd-service-registry_bbc930685b444ab7b382e919df21c501/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-6zqi0xt9\n         cwd: /tmp/pip-install-xgj4jp82/nd-service-registry_bbc930685b444ab7b382e919df21c501/\n    Complete output (47 lines):\n    WARNING: The wheel package is not available.\n    WARNING: The wheel package is not available.\n    WARNING: The wheel package is not available.\n    WARNING: The wheel package is not available.\n      ERROR: Command errored out with exit status 1:\n       command: /usr/bin/python3.8 -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-wheel-tvg_7zw5/future_0e8eefad65b443cf80f6245d9f67cd4e/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-wheel-tvg_7zw5/future_0e8eefad65b443cf80f6245d9f67cd4e/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' bdist_wheel -d /tmp/pip-wheel-6gcs8yk9\n           cwd: /tmp/pip-wheel-tvg_7zw5/future_0e8eefad65b443cf80f6245d9f67cd4e/\n      Complete output (6 lines):\n      usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]\n         or: setup.py --help [cmd1 cmd2 ...]\n         or: setup.py --help-commands\n         or: setup.py cmd --help\n    \n      error: invalid command 'bdist_wheel'\n      ----------------------------------------\n      ERROR: Failed building wheel for future\n    ERROR: Failed to build one or more wheels\n    Traceback (most recent call last):\n      File \"/usr/lib/python3.8/site-packages/setuptools/installer.py\", line 75, in fetch_build_egg\n        subprocess.check_call(cmd)\n      File \"/usr/lib/python3.8/subprocess.py\", line 364, in check_call\n        raise CalledProcessError(retcode, cmd)\n    subprocess.CalledProcessError: Command '['/usr/bin/python3.8', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmps78u33b3', '--quiet', 'future']' returned non-zero exit status 1.\n    \n    The above exception was the direct cause of the following exception:\n    \n    Traceback (most recent call last):\n      File \"<string>\", line 1, in <module>\n      File \"/tmp/pip-install-xgj4jp82/nd-service-registry_bbc930685b444ab7b382e919df21c501/setup.py\", line 142, in <module>\n        setup(\n      File \"/usr/lib/python3.8/site-packages/setuptools/__init__.py\", line 152, in setup\n        _install_setup_requires(attrs)\n      File \"/usr/lib/python3.8/site-packages/setuptools/__init__.py\", line 147, in _install_setup_requires\n        dist.fetch_build_eggs(dist.setup_requires)\n      File \"/usr/lib/python3.8/site-packages/setuptools/dist.py\", line 721, in fetch_build_eggs\n        resolved_dists = pkg_resources.working_set.resolve(\n      File \"/usr/lib/python3.8/site-packages/pkg_resources/__init__.py\", line 766, in resolve\n        dist = best[req.key] = env.best_match(\n      File \"/usr/lib/python3.8/site-packages/pkg_resources/__init__.py\", line 1051, in best_match\n        return self.obtain(req, installer)\n      File \"/usr/lib/python3.8/site-packages/pkg_resources/__init__.py\", line 1063, in obtain\n        return installer(requirement)\n      File \"/usr/lib/python3.8/site-packages/setuptools/dist.py\", line 780, in fetch_build_egg\n        return fetch_build_egg(self, req)\n      File \"/usr/lib/python3.8/site-packages/setuptools/installer.py\", line 77, in fetch_build_egg\n        raise DistutilsError(str(e)) from e\n    distutils.errors.DistutilsError: Command '['/usr/bin/python3.8', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmps78u33b3', '--quiet', 'future']' returned non-zero exit status 1.\n    ----------------------------------------\n\u001b[0m\u001b[91mWARNING: Discarding https://files.pythonhosted.org/packages/c2/31/5428bd9182db105ed9f822f03b3f1aaf4e64d6224b4294f137b943d072be/nd_service_registry-1.1.0.tar.gz#sha256=9b1e3cb8ae2730229aea6fef791f4c70b46fcc2ad1351091150b05db139fda6a (from https://pypi.org/simple/nd-service-registry/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\n\u001b[0m  Downloading nd_service_registry-1.0.0.tar.gz (44 kB)\n\u001b[91m    ERROR: Command errored out with exit status 1:\n     command: /usr/bin/python3.8 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-xgj4jp82/nd-service-registry_6d6f33a4fe24439486e2fa9431b6906e/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-xgj4jp82/nd-service-registry_6d6f33a4fe24439486e2fa9431b6906e/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-ujdshfya\n         cwd: /tmp/pip-install-xgj4jp82/nd-service-registry_6d6f33a4fe24439486e2fa9431b6906e/\n    Complete output (5 lines):\n    Traceback (most recent call last):\n      File \"<string>\", line 1, in <module>\n      File \"/tmp/pip-install-xgj4jp82/nd-service-registry_6d6f33a4fe24439486e2fa9431b6906e/setup.py\", line 27, in <module>\n        execfile(os.path.join(PACKAGE, 'version.py'))  # set __version__\n    NameError: name 'execfile' is not defined\n    ----------------------------------------\n\u001b[0m\u001b[91mWARNING: Discarding https://files.pythonhosted.org/packages/f7/0c/908e08d48edc3268fa0f6c5da7f92f6bb9305f78f8645951200e75c53233/nd_service_registry-1.0.0.tar.gz#sha256=5dc8263c3a21344d79ea477d45964ce9e21a429f67741cbe615926e27b9c98a0 (from https://pypi.org/simple/nd-service-registry/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\n\u001b[0m\u001b[91mERROR: Could not find a version that satisfies the requirement nd_service_registry>=1.0.0 (from versions: 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9, 0.3.0, 0.3.1, 0.3.2, 1.0.0, 1.1.0)\n\u001b[0m\u001b[91mERROR: No matching distribution found for nd_service_registry>=1.0.0\n\u001b[0m\u001b[91mWARNING: You are using pip version 21.1.1; however, version 22.3.1 is available.\nYou should consider upgrading via the '/usr/bin/python3.8 -m pip install --upgrade pip' command.\n\u001b[0mRemoving intermediate container 206de030354a\n","stderr":"The command '/bin/sh -c pip3 install -r /app/requirements.test.txt -r /app/requirements.txt' returned a non-zero code: 1\n"},"endTime":1672423899910}